lib/index.d.ts

```diff
@@ -1,7 +1,13 @@
+// Type definitions for panel
```

Comment:
I'd love for our tests fixtures in https://github.com/mixpanel/panel/tree/master/test/fixtures to be typechecked via typescript's `allowJs` and `checkJs` features. You don't have to do it in this PR but it would be great validation that the .d.ts file is doing the right thing and works with existing code.